lib/utils/generate_pokemon_image.dart

```diff
@@ -0,0 +1,4 @@
+String generateImage(int index) {
+  index += 1;
+  return "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/$index.png";
```

Comment:
This is what I told you last time about putting this into constants so that you can just use it like 
k.pokeImage.replase(with index)

Reply:
Okay I understand now hehe